sklearn/ensemble/forest.py

```diff
@@ -1220,11 +1220,18 @@ class RandomForestRegressor(ForestRegressor):
     search of the best split. To obtain a deterministic behaviour during
     fitting, ``random_state`` has to be fixed.
 
+    The default value ``max_features="auto"`` uses ``n_features`` 
+    rather than ``n_features / 3``. The latter was originally suggested in
+    [1], whereas the former was more recently justified empirically in [2].
+
     References
     ----------
 
     .. [1] L. Breiman, "Random Forests", Machine Learning, 45(1), 5-32, 2001.
 
+    .. [2] P. Geurts, D. Ernst., and L. Wehenkel, "Extremely randomized trees",
```

Comment:
Please take care for your lines to not exceed 79 characters. Thanks

Reply:
No problem. I copied the reference line from elsewhere in the `forest.py` file. I can update the reference line in both places if you'd like.

Reply:
Please do, right now Travis CI fails do to flake8, and we can't merge it until it's fixed. The critical part is fixing the lines you added / edited (flake8 in other parts of the exiting code base are fine).